java-client-generated/src/main/java/org/ergoplatform/explorer/client/model/OutputInfo.java

```diff
@@ -256,23 +251,23 @@ public void setAssets(java.util.List<AssetInstanceInfo> assets) {
     this.assets = assets;
   }
 
-//  public OutputInfo additionalRegisters(AdditionalRegisters additionalRegisters) {
-//    this.additionalRegisters = additionalRegisters;
-//    return this;
-//  }
-
-//   /**
-//   * Get additionalRegisters
-//   * @return additionalRegisters
-//  **/
-//  @Schema(required = true, description = "")
-//  public AdditionalRegisters getAdditionalRegisters() {
-//    return additionalRegisters;
-//  }
-//
-//  public void setAdditionalRegisters(AdditionalRegisters additionalRegisters) {
-//    this.additionalRegisters = additionalRegisters;
-//  }
+  public OutputInfo additionalRegisters(AdditionalRegisters additionalRegisters) {
+    this.additionalRegisters = additionalRegisters;
+    return this;
+  }
+
+   /**
+   * Get additionalRegisters
+   * @return additionalRegisters
+  **/
+  @Schema(required = true, description = "")
```

Comment:
shouldn't it be required = false?

Reply:
Explorer API always returns it, but empty.